common/src/main/java/org/figuramc/figura/lua/api/ConfigAPI.java

```diff
@@ -79,6 +79,19 @@ private Path getPath() {
         }
     }
 
+    private void writeWithRetries(Path path, byte[] data, int retries) {
```

Comment:
I'm here to play catch, not write a 30-page paper on the intricacies of the game we know as 'catch'.

```suggestion
    private void writeWithRetries(Path path, byte[] data, int retries) throws Exception {
```